magefile.go

```diff
@@ -679,7 +679,7 @@ func Install() {
 		// Removing the file first clears the cache so that we don't run into "zsh: killed MIXIN..."
 		// See https://stackoverflow.com/questions/67378106/mac-m1-cping-binary-over-another-results-in-crash
 		// See https://openradar.appspot.com/FB8914231
-		mgx.Must(os.Remove(filepath.Join(destDir, mixin+xplat.FileExt())))
+		//mgx.Must(os.Remove(filepath.Join(destDir, mixin+xplat.FileExt())))
```

Comment:
This line was there for an issue on M1 Macs where there would be an error about the executable not being signed properly and wouldn't execute. Are you able to confirm you're not able to reproduce that error?
If so, you can just delete this line and all the comments addressing it :)

Reply:
Hey @schristoff  give me some time will confirm it and drop a message here